SWProxy.py

```diff
@@ -225,45 +225,48 @@ def parse_pcap(filename):
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description='SWParser')
-    parser.add_argument('-d', '--debug', action="store_true", default=False)
-    parser.add_argument('-g', '--no-gui', action="store_true", default=False)
-    parser.add_argument('-p', '--port', type=int, default=8080)
-    options = parser.parse_args()
+    if len(sys.argv) >= 2 and sys.argv[1].endswith('.pcap'):
+        parse_pcap(sys.argv[1])
```

Comment:
I think before calling parse_pcap, you should set up the logger as well and call the 'print get_usage_text()' as well.